src/main/java/task.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+
+public class task {
+    private int mark;
+    private String line;
+    ArrayList<String> namelist=new ArrayList<>();
+    ArrayList<Character> donelist=new ArrayList<>();
+    public task(){
+        this(0,"null");
+    }
+    public task(int mark, String line){
+        this.mark=mark;
+        this.line=line;
+    }
+    public void getList(){
+      System.out.println("Here are the tasks in your list:");
+        for(int ay=0;ay<namelist.size();ay++){
+            System.out.println((ay+1)+". ["+donelist.get(ay)+"] "+namelist.get(ay));
+        }
+        //return 1;
+    }
+    public void setList(int mark){
+        this.mark=mark-1;//check
+        donelist.set(this.mark, 'X');
```

Comment:
is it better to extract the 'X' out and code it as a symbol constant like private static final String MARK_AS_DONE = 'X'